draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
+                                        id BIGSERIAL NOT NULL,
+                                        condition TEXT NOT NULL,
+                                        flow_type TEXT NOT NULL,
+                                        event_type TEXT NOT NULL,
+                                        threshold_value FLOAT NOT NULL,
+                                        target_uri TEXT NOT NULL,
+                                        cooldown_period INTEGER NOT NULL DEFAULT 0,
+                                        created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                                        updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                                        status TEXT NOT NULL,
+                                        owner_dev_license_address BYTEA NOT NULL,
+
+                                        CONSTRAINT webhooks_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS events (
+                                      id BIGSERIAL NOT NULL,
+                                      webhook_id BIGSERIAL REFERENCES webhooks(id) NOT NULL,
+                                      snapshot_data TEXT NOT NULL,
+                                      http_response_code INTEGER,
+                                      created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+                                      CONSTRAINT events_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS event_cooldowns (
```

Comment:
I feel like this could be merged with the above event logs table, it basically has the same columns, or like just add the last_triggered_at to event_logs